src/lib/patternfly-4-cockpit.scss

```diff
@@ -0,0 +1,14 @@
+/* Here we need to import patternfly-base.scss but the fonts file from PF4 is containing
+ * urls to font formats we are not interested in including.
+ * @import '@patternfly/patternfly/patternfly-base.scss';
```

Comment:
That comment doesn't really explain the conclusion -- either this import is the desired form, and the variable settings and individual imports below are a workaround, then it should point that out. Or this generally isn't meant to work, then let's drop the commented-out `@import` completely.

I. e. the code below makes it look like you could first define the three variables and then import patternfly-base.scss?

Reply:
`@import '@patternfly/patternfly/patternfly-base.scss'` is what PF4 exposes for us normally.
It's consisting from all the imports I do, plus one more that I replaced with the `@import fonts` line.
The line that is missing is:
`@import "../../node_modules/@patternfly/patternfly/fonts";`